var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -210,17 +216,17 @@ def install(self, spec, prefix):
         #
         # Below we try to match the entire HDF5_LIB line and substitute
         # with the list of libraries that needs to be linked.
-        if self.spec.variants['hdf5'].value != 'none':
+        if spec.variants['hdf5'].value != 'none':
             make_inc = join_path(self.stage.source_path, 'make.inc')
             hdf5_libs = ' '.join(spec['hdf5:hl,fortran'].libs)
             filter_file(r'HDF5_LIB([\s]*)=([\s\w\-\/.,]*)',
                         'HDF5_LIB = {0}'.format(hdf5_libs),
                         make_inc)
-            if self.spec.variants['hdf5'].value == 'serial':
+            if spec.variants['hdf5'].value == 'serial':
                 # Note that there is a benign side effect with this filter
                 # file statement. It replaces an instance of MANUAL_DFLAGS
                 # that is a comment in make.inc.
-                filter_file(r'MANUAL_DFLAGS([\s]*)=([\s]*)',
+                filter_file(r'MANUAL_DFLAGS([\s]*)=([\s]*)'
```

Comment:
Did you intend to remove the `,` here?

Reply:
thanks for catching that -- that's what I get for working late in the evening.